src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinHttpWebSocketCallback.cs

```diff
@@ -357,20 +357,6 @@ private static void OnWebSocketHandleClosing(WinHttpWebSocketState state)
                     break;
             }
         }
-
-        private static void OnWebSocketSecureFailure(WinHttpWebSocketState state, uint flags)
```

Comment:
Please don't delete this method (OnWebSocketSecureFailure).  There are active issues associated with this code.  And this code will be used when those issues are worked on.